pkg/diagnostics/network/objects.go

```diff
@@ -13,21 +13,23 @@ import (
 )
 
 const (
-	busyboxImage               = "docker.io/busybox"
+	diagnosticsImage           = "openshift/diagnostics-deployer"
```

Comment:
Is this affected by imageConfig.format? If not how should this work for OSE where the images are usually 'openshift3/ose-${component}:${version}' ?